DESCRIPTION

```diff
@@ -30,7 +30,7 @@ Imports:
     withr
 Suggests: 
     covr,
-    testthat
+    testthat (>= 2.0.0)
```

Comment:
because I'm starting to consult the `TESTTHAT` envvar

Reply:
Should probably use some `expect_known_output()` too